### PR TITLE
fix docstrings: paddle.nn.Conv3DTranspose, etc.

### PR DESCRIPTION
**Pull request by LearningPawn**
### PR types

Bug fixes
### PR changes

Docs
### Describe

1. 修复了 python/paddle/nn/layer/conv.py 中出现的 Tex 语法问题；
2. 为 python/paddle/nn/functional/conv.py 修复 Tex 语法问题，修改 note 及参数说明格式，尝试添加论文链接；
3. 修改 python/paddle/nn/layer/loss.py、python/paddle/nn/functional/loss.py 中参数说明的格式；
4. 修改 python/paddle/nn/layer/common.py、python/paddle/nn/functional/common.py 中参数说明的格式，添加 optional 说明。

涉及到的 API 有

- paddle.nn.Conv3DTranspose
- paddle.nn.functional.conv3d_transpose
- paddle.nn.CTCLoss
- paddle.nn.functional.ctc_loss
- paddle.nn.Dropout
- paddle.nn.functional.dropout
- paddle.nn.Dropout2D
- paddle.nn.functional.dropout2d
- paddle.nn.Dropout3D
- paddle.nn.functional.dropout3d

对应的中文文档修复为：https://github.com/PaddlePaddle/docs/pull/5526

### CODE REVIEW

File: python/paddle/nn/layer/loss.py [hunk 1]
```diff
@@ -1042,7 +1042,7 @@ class CTCLoss(Layer):
         labels (Tensor): The ground truth sequence with padding, which must be a 3-D Tensor. The tensor shape is [batch_size, max_label_length], where max_label_length is the longest length of label sequence. The data type must be int32.
```
Comment (Ligoml) [inline python/paddle/nn/layer/loss.py]: image

这里可能需要使用md格式，加上 `-`来分点

Comment (Ligoml) [reply python/paddle/nn/layer/loss.py]: 预览可以在commit后面加上 `test=docs_preview`，具体用法可参考 [飞桨文档预览工具](https://github.com/PaddlePaddle/docs/wiki/%E9%A3%9E%E6%A1%A8%E6%96%87%E6%A1%A3%E9%A2%84%E8%A7%88%E5%B7%A5%E5%85%B7)

File: python/paddle/nn/layer/common.py [hunk 2]
```diff
@@ -710,20 +710,20 @@ class Dropout(Layer):
...
 
     Parameters:
-        p (float|int): Probability of setting units to zero. Default: 0.5
-        axis (int|list|tuple): The axis along which the dropout is performed. Default None.
+        p (float|int, optional): Probability of setting units to zero. Default: 0.5
+        axis (int|list|tuple, optional): The axis along which the dropout is performed. Default: None.
         mode(str, optional): ['upscale_in_train'(default) | 'downscale_in_infer']
 
-                               1. upscale_in_train(default), upscale the output at training time
+                               1. upscale_in_train (default), upscale the output at training time
```
Comment (Ligoml) [inline python/paddle/nn/layer/common.py:717]: image

预览看起来报错了

**Comment by LearningPawn** [comment]
@Ligoml 您辛苦了，我已经根据您的 Review 意见进行了修改：
1. [CTCLoss](http://preview-paddle-pr-49149.paddle-docs-preview.paddlepaddle.org.cn/documentation/docs/en/api/paddle/nn/CTCLoss_en.html) 的 Shapes: 已经使用 `-` 修改；
2. [Dropout](http://preview-paddle-pr-49149.paddle-docs-preview.paddlepaddle.org.cn/documentation/docs/en/api/paddle/nn/Dropout_en.html) 的 Parameters 现已无报错。

**Review by Ligoml — DISMISSED** [review]
good job！LGTM for docs

File: python/paddle/nn/functional/conv.py [hunk 3]
```diff
@@ -1652,36 +1652,36 @@ def conv3d_transpose(
...
-          The conv3d_transpose can be seen as the backward of the conv3d. For conv3d,
-          when stride > 1, conv3d maps multiple input shape to the same output shape,
-          so for conv3d_transpose, when stride > 1, input shape maps multiple output shape.
-          If output_size is None, :math:`H_{out} = H^\prime_{out}, :math:`H_{out} = \
-          H^\prime_{out}, W_{out} = W^\prime_{out}`; else, the :math:`D_{out}` of the output
-          size must between :math:`D^\prime_{out}` and :math:`D^\prime_{out} + strides[0]`,
-          the :math:`H_{out}` of the output size must between :math:`H^\prime_{out}`
-          and :math:`H^\prime_{out} + strides[1]`, and the :math:`W_{out}` of the output size must
-          between :math:`W^\prime_{out}` and :math:`W^\prime_{out} + strides[2]`.
+    .. note::
```
Comment (Ligoml) [inline python/paddle/nn/functional/conv.py]:
```suggestion
    Note:
```

Comment (Ligoml) [reply python/paddle/nn/functional/conv.py]: 其实两种方式都可以啦

**Review by Ligoml — DISMISSED** [review]

**Review by Ligoml — DISMISSED** [review]
LGTM for docs
检查预览后合入

**Review by Ligoml — APPROVED** [review]
